Docs/minutes-02.md

```diff
@@ -0,0 +1,40 @@
+Everyone do our parts and then get them done by Wednesday so that we can go over them in the lab.
+lab to make sure we're on the right track.
+
+Make sure we all do a little
+
+
+
+Make it so that we start up the db with the students and info already there
+
+Jack's issue is a great first issue!
+
+Need to start work on the presentation
+- do research on acceptance testing so that we can talk about that 
+  *I'll make a new doc for talking about the research in the same shared doc*
+  
+  
+  
+To run, comment out first 4 lines in the applications properties, and make sure that you
+Install postgres so that you can run it and it's on https://peerassessmenttool-jora.herokuapp.com/jorahome
+
+Now we're gonna talk about what we all need to do:
+On tuesday get as much done as we can.
+Groups - Riley
+
+Finalize schema.  
+
+Jack got rid of the student list, it was messing up 
```

Comment:
Mayber say, "Jack got rid of the student list in the Group class", it was messing up

Reply:
> Mayber say, "Jack got rid of the student list in the Group class", it was messing up

```suggestion
Jack got rid of the student list in the Group class, it was messing up 
```